packages/core/admin/package.json

```diff
@@ -49,6 +57,7 @@
     "@strapi/icons": "1.11.0",
     "@strapi/permissions": "4.14.0",
     "@strapi/provider-audit-logs-local": "4.14.0",
+    "@strapi/types": "4.14.0",
```

Comment:
@innerdvations @Convly  eslint is asking me to put @strapi/types into `dependencies` and not `devDependencies`. Is that intended?

Reply:
That's weird, is that a warning or an error?

Reply:
eslint error